readme.md

```diff
@@ -18,6 +18,9 @@ var isAbsoluteUrl = require('is-absolute-url');
 isAbsoluteUrl('http://sindresorhus.com/foo/bar');
 //=> true
 
+isAbsoluteUrl('//sindresorhus.com');
+//=> false
```

Comment:
Not sure if I'm the correct person to explain why it's not. I just created this PR based on the discussion in #2.